packages/meditrak-app/app/assessment/DumbSurveyScreen.js

```diff
@@ -28,13 +21,14 @@ import {
 } from '../widgets';
 import { SurveyTableOfContents } from './SurveyTableOfContents';
 import { THEME_COLOR_ONE } from '../globalStyles';
+import { HeaderLeftButton } from '../navigation/HeaderLeftButton';
 
 const LENGTH_OF_TRANSITION = 300;
 
 export class DumbSurveyScreen extends React.Component {
   static navigationOptions = ({ navigation }) => ({
-    title: navigation.state.params.headerLabel,
-    headerBackImage: require('../images/x.png'),
+    headerTitle: navigation.state.params.headerLabel,
+    headerLeft: () => <HeaderLeftButton source={require('../images/x.png')}></HeaderLeftButton>,
```

Comment:
Could this be self-closing?
```suggestion
    headerLeft: () => <HeaderLeftButton source={require('../images/x.png')} />,
```